sourced/ml/__main__.py

```diff
@@ -47,6 +46,23 @@ def add_default_args(my_parser):
     # Create and construct subparsers
     subparsers = parser.add_subparsers(help="Commands", dest="command")
 
+    repos2df_parser = subparsers.add_parser(
+        "repos2df", help="Convert source code to weighted sets.")
+    repos2df_parser.set_defaults(handler=repos2df_entry)
+    add_default_args(repos2df_parser)
+    add_engine_args(repos2df_parser)
+    repos2df_parser.add_argument(
+        "--docfreq", required=True,
+        help="[OUT] The path to the OrderedDocumentFrequencies model.")
```

Comment:
Does it have to be ordered? I would add `--ordered` which forces the result to be ordered, and False by default.